schemes/global.tt

```diff
@@ -102,6 +102,23 @@ the same terms as Perl itself.  For a copy of the license, please reference
 [%- END -%]
 [%- END -%]
 
+[%- BLOCK block.msgs -%]
+[%# sections.errors are instances of DW::FormErrors %]
```

Comment:
Is this comment true in the block.msgs too?

Reply:
oops, nope, that's what I get for copying the formerror block wholesale to figure out how to stick it into all templates.